concordia/settings_prod.py

```diff
@@ -0,0 +1,21 @@
+from .settings_template import *
+ 
+LOGGING["handlers"]["stream"]["level"] = "INFO"
+LOGGING["handlers"]["file"]["level"] = "INFO"
+LOGGING["handlers"]["file"]["filename"] = "./logs/concordia-web.log"
+LOGGING["handlers"]["celery"]["level"] = "INFO"
+LOGGING["handlers"]["celery"]["filename"] = "./logs/concordia-celery.log"
+LOGGING["loggers"]["django"]["level"] = "INFO"
+LOGGING["loggers"]["celery"]["level"] = "INFO"
+
+DJANGO_SECRET_KEY = "np*no_id-t8*if4*a(pz(n(rk5#(rlyn-41jvp5u%*ij&5u-3x"
```

Comment:
I generally like setting these to something which makes it obvious that this is not the value which will actually be deployed